tagging/utils/docker_runner.py

```diff
@@ -22,11 +22,10 @@ def __init__(
         self.docker_client: docker.DockerClient = docker_client
 
     def __enter__(self) -> Container:
-        LOGGER.info(f"Creating container for image {self.image_name} ...")
+        LOGGER.info(f"Creating a container for the image: {self.image_name} ...")
+        default_kwargs = {"detach": True, "tty": True}
```

Comment:
This class is more-or-less like TrackedContainer, so let's have same options here